src/service/fileMaker.ts

```diff
@@ -4,20 +4,69 @@ import * as fs from '../util/fs';
 import { log } from '../util/logger';
 import { interpolate } from '../util/string';
 import { getTimestampString } from '../util/ts';
+import MakeOptions from '../domain/MakeOptions';
 import Configuration from '../domain/Configuration';
 import FileExtensions from '../enum/FileExtensions';
 import { getMigrationPath } from '../migration/service/knexMigrator';
 
 const MIGRATION_TEMPLATE_PATH = path.resolve(__dirname, '../../assets/templates/migration');
 const CREATE_TABLE_CONVENTION = /create_(\w+)_table/;
+const SOURCE_TYPE_STUBS = {
```

Comment:
Instead of using arrays can we use key value pairs instead?

```
sql: {
  up: 'create_up.stub',
  down: 'create_down.stub'
},
javascript: {
  create: 'create_js.stub',
  alter: 'update_js.stub'
}
...
```

Reply:
Anyways it is not a big thing. :D

Reply:
The filename itself explains most of the thing. So resolving this.